web-tck/src/main/java/ws/ament/hammock/web/tck/ServletTest.java

```diff
@@ -40,7 +39,7 @@
 @RunWith(Arquillian.class)
 public abstract class ServletTest {
     public static JavaArchive createArchive(Class<?>... classes) {
-        String property = System.getProperty(Bootstrapper.class.getName(), Weld3Bootstrapper.class.getName());
+        String property = System.getProperty(Bootstrapper.class.getName(), "ws.ament.hammock.bootstrap.weld3.Weld3Bootstrapper");
```

Comment:
Now that I'm seeing this, this property can be removed.  Can you try without it?

Reply:
Yes works with mvn test.
Only when running test by ide it fails.